### ISSUE TITLE
protontricks fails to find pressure-vessel-launch

### ISSUE THREAD

**Issue by Photon89** [Matoking/protontricks#153]
**Describe the bug**
I am trying to install vcrun2013 for Assetto Corsa but the installation fails (output see last section).

**To Reproduce**
1. Run command `protontricks 244210 vcrun2013`
2. Command fails and error is displayed (as described above)

**Expected behavior**
The installation should finish successfully

**System (please complete the following information):**
 - Distro: Arch Linux
 - Protontricks installation method: AUR package
 - Protontricks version: 1.8.1
 - Steam version: No beta participation
 - Systemwide installed Wine version: 7.7
 
**Additional context**

I use Proton GE and an external Steam library. Indeed,  pressure-vessel-launch is not present where protontricks searches for it:

```
$ ls /media/Daten/Steam/steamapps/common/SteamLinuxRuntime_soldier/pressure-vessel/bin/
pressure-vessel-adverb         pv-bwrap
pressure-vessel-locale-gen     steam-runtime-launch-client
pressure-vessel-test-ui        steam-runtime-launcher-service
pressure-vessel-try-setlocale  steam-runtime-launch-options
pressure-vessel-unruntime      steam-runtime-system-info
pressure-vessel-wrap
```

**If the error happens when trying to run a Protontricks command, run the command again using the `-v` flag and copy the output!**
```

$ LANG=C protontricks -v 244210 vcrun2013
protontricks (INFO): Found Steam directory at /home/photon/.steam/steam. You can also define Steam directory manually using $STEAM_DIR
protontricks (INFO): Using default Steam Runtime at /home/photon/.steam/root/ubuntu12_32/steam-runtime
protontricks (INFO): WINETRICKS environment variable is not available. Searching from $PATH.
protontricks (INFO): Found 2 Steam library folders
protontricks (INFO): Currently logged-in Steam user: photon89
protontricks (INFO): Couldn't find custom shortcuts. Maybe none have been created yet?
protontricks (INFO): User has configured app Proton version (CompatToolMapping): GE-Proton7-16
protontricks (INFO): User has configured default Proton version (CompatToolMapping): proton_42
protontricks (INFO): Found active custom compatibility tool: GE-Proton7-16
protontricks (INFO): Active compatibility tool is a Proton installation
protontricks (INFO): Using separately installed Steam Runtime: Steam Linux Runtime - Soldier
protontricks (INFO): Running Steam Runtime using bwrap containerization.
If any problems arise, please try running the command again using the `--no-bwrap` flag and make an issue report if the problem only occurs when bwrap is in use.
protontricks (INFO): Created Steam Runtime Wine binary directory at /home/photon/.cache/protontricks/proton/GE-Proton7-16/bin
protontricks (INFO): WINE environment variable is not available. Setting WINE environment variable to Proton bundled version.
protontricks (INFO): WINESERVER environment variable is not available. Setting WINESERVER environment variable to Proton bundled version
protontricks (INFO): Starting bwrap launcher process: /home/photon/.cache/protontricks/proton/GE-Proton7-16/bin/bwrap-launcher
protontricks (INFO): Starting wineserver keepalive process: /home/photon/.cache/protontricks/proton/GE-Proton7-16/bin/wineserver-keepalive
protontricks (INFO): Attempting to run command ['/usr/sbin/winetricks', 'vcrun2013']
protontricks - bwrap-launcher 4177: Following directories will be mounted inside container: /boot /etc /home /lost+found /media /mnt /opt /root /sbin /srv /tmp /run/media
protontricks - wineserver-keepalive 4178: Starting wineserver-keepalive process...
protontricks - bwrap-launcher 4177: Using temporary directory: /tmp/protontricks-s1w15q_1
------------------------------------------------------
warning: Your version of wine  is no longer supported upstream. You should upgrade to 5.x
------------------------------------------------------
Executing mkdir -p /media/Daten/Steam/steamapps/compatdata/244210
protontricks - wineserver-keepalive 4178: Cleanup finished, goodbye!
------------------------------------------------------
warning: You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
------------------------------------------------------
WINEPREFIX INFO:
Drive C: total 36
drwxr-xr-x  9 photon users 4096 Jun  4 21:46 .
drwxr-xr-x  4 photon users 4096 Jun  4 23:01 ..
drwxr-xr-x  6 photon users 4096 Jun  4 21:08 Program Files
drwxr-xr-x  8 photon users 4096 Jun  4 21:32 Program Files (x86)
drwxr-xr-x  4 photon users 4096 Jun  4 21:39 ProgramData
drwxr-xr-x  2 photon users 4096 Jun  4 22:47 openxr
drwxr-xr-x  4 photon users 4096 Jun  4 21:08 users
drwxr-xr-x  3 photon users 4096 Jun  4 21:32 vrclient
drwxr-xr-x 22 photon users 4096 Jun  4 21:42 windows

Registry info:
/media/Daten/Steam/steamapps/compatdata/244210/pfx/system.reg:#arch=win64
/media/Daten/Steam/steamapps/compatdata/244210/pfx/user.reg:#arch=win64
/media/Daten/Steam/steamapps/compatdata/244210/pfx/userdef.reg:#arch=win64
------------------------------------------------------
------------------------------------------------------
warning: /home/photon/.cache/protontricks/proton/GE-Proton7-16/bin/wine cmd.exe /c echo '%AppData%' returned empty string, error message "protontricks - wine 4546: Checking for running wineserver instance
protontricks - wine 4546: Setting default env: WINEFSYNC=1
protontricks - wine 4546: Setting default env: WINEESYNC=1
protontricks - wine 4546: Starting Wine process using 'pressure-vessel-launch'
/home/photon/.cache/protontricks/proton/GE-Proton7-16/bin/wine: line 155: /media/Daten/Steam/steamapps/common/SteamLinuxRuntime_soldier/pressure-vessel/bin/pressure-vessel-launch: No such file or directory" 
------------------------------------------------------
protontricks (INFO): Terminating wineserver keepalive process 4178
protontricks (INFO): Terminating launcher process 4177
protontricks (INFO): Launcher process terminated
```

**Comment by hbjngf** [comment]
Same issue on Rocksmith 2014 as of today

**Comment by Photon89** [comment]
I think, this is not game specific, actually. Maybe it is a problem with Proton GE? Do you also use Proton GE? And do you use an external Steam library?

**Comment by hbjngf** [comment]
Steam library is on the same drive, all in standard install locations, was able to use winetricks directly by setting the WINEPREFIX and WINE variables to the games prefix and proton GE manually

**Comment by hbjngf** [comment]
You'd need to change the APPID and username, but for Proton 7.18 GE by default its
WINEPREFIX=/home/USERNAMEHERE/.steam/steam/steamapps/compatdata/APPIDHERE/pfx WINE=/home/jesse/.steam/steam/compatibilitytools.d/GE-Proton7-18/files/bin/wine winetricks

**Comment by Photon89** [comment]
Hmm, this seems to have worked, but Assetto Corsa still reports

```
643.162:036c:0370:err:module:import_dll Library MSVCR120.dll (which is needed by L"C:\\windows\\system32\\MSVCP120.dll") not found
643.162:036c:0370:err:module:import_dll Library MSVCP120.dll (which is needed by L"Z:\\media\\Daten\\Steam\\steamapps\\common\\assettocorsa\\acs.exe") not found
643.163:036c:0370:err:module:import_dll Library MSVCR120.dll (which is needed by L"Z:\\media\\Daten\\Steam\\steamapps\\common\\assettocorsa\\acs.exe") not found
```

which should have been fixed by the vcrun2013 installation...

**Comment by Matoking** [comment]
The "cannot find `steam-runtime-launch-client`" issue should be fixed starting with 1.8.2. Please try updating Protontricks first.

Since you've installed from AUR you're better off installing `protontricks-git` instead for the bug fix. The package maintainer appears to be inactive, and the `protontricks` package is still on 1.8.1.

**Comment by Photon89** [comment]
Using the -git 1.8.2 version works, thanks!
